code/__DEFINES/dcs/signals/signals_datum.dm

```diff
@@ -31,3 +31,17 @@
 #define COMSIG_MERGER_REMOVING "comsig_merger_removing"
 /// Called on the merger after finishing a refresh: (list/leaving_members, list/joining_members)
 #define COMSIG_MERGER_REFRESH_COMPLETE "comsig_merger_refresh_complete"
+
+// Gas mixture signals
+/// From /datum/gas_mixture/proc/merge: ()
+#define COMSIG_GASMIX_MERGING "comsig_gasmix_merged"
+/// From /datum/gas_mixture/proc/remove: ()
+#define COMSIG_GASMIX_REMOVING "comsig_gasmix_removing"
+/// From /datum/gas_mixture/proc/react: ()
+#define COMSIG_GASMIX_REACTING "comsig_gasmix_reacting"
+
+// Modular computer's file signals. Used to tell the program datum something is going on.
```

Comment:
Why do you have this used bit of the comment?

Reply:
Just to let people know that the program is the signal target. I suppose i'll change "used" to something else.